### DISCUSSION TITLE
Switch Pixelate mode

### DISCUSSION THREAD

**Discussion by iseries** [flameshot-org/flameshot#3144] [Q&A]
Hello,

there is a feature called Pixelate. Somehow i switched to a blur mode while using it. How can i switch back?

App is:
Flameshot v11.0.0 (11.0.0-2 Debian)
Compiled with Qt 5.15.2

System is:
Linux x1 5.19.0-35-generic #36~22.04.1-Ubuntu SMP PREEMPT_DYNAMIC Fri Feb 17 15:17:25 UTC 2 x86_64 x86_64 x86_64 GNU/Linux

Thanks

**Comment by iseries** [comment]
solved it. by scrolling.

**Reply by mmahmoudian** [reply]
Basically the idea is that blur is the smallest pixel size (not in reality, but that is how we think about it). By scrolling you can change the tool size, and in case of pixelate tool, you can change the pixel size.

As a side note: if you are using it to hide sensitive content, choose a pixel size that at least is as high as your text's line height.

Another side note: you can draw anything in Flameshot, and then deselect the tool and select each of those object and change their properties including size, color, position, layer order, ...
